utils/http-calls.js

```diff
@@ -0,0 +1,11 @@
+import { SERVER_URL } from '../config';
+
+export async function apiFetch(endpoint) {
+    let response = await fetch(`${SERVER_URL}/api/${endpoint}`);
+
+    if (response.ok) {
+        return response.json()
+        
+    } else 
+        throw `An error when fetching from /api/${endpoint} - ${response.text}`
```

Comment:
Why aren't you using `.then()` and `.catch()`?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function#rewriting_a_promise_chain_with_an_async_function